lib/workers/repository/process/lookup/filter-checks.ts

```diff
@@ -53,6 +54,16 @@ export async function filterInternalChecks(
       );
       // Apply packageRules in case any apply to updateType
       releaseConfig = applyPackageRules(releaseConfig, 'update-type');
+
+      const updatedCandidateRelease = await postprocessRelease(
+        releaseConfig,
+        candidateRelease,
+      );
+      if (!updatedCandidateRelease) {
+        continue;
+      }
```

Comment:
Can we add these to a new/optional list like `rejectedReleleases` so there's visibility in `packageFiles with updates`?